src/applications/applications.controller.ts

```
@@ -67,7 +73,20 @@ export class ApplicationsController {
+    const parser = new UAParser();
+
+    const userAgent = req.headers['user-agent'] || '';
+    const uaResult = parser.setUA(userAgent).getResult();
+    const os = [uaResult.os.name, uaResult.os.version].filter(Boolean).join(' ');
+    const browser = [uaResult.browser.name, uaResult.browser.version].filter(Boolean).join(' ');
+    const updatedBy = req.mw_userid;
+    const ip = Array.isArray(req.headers['x-forwarded-for'])
+      ? req.headers['x-forwarded-for'][0]
+      : req.headers['x-forwarded-for'] || req.socket.remoteAddress || '';
+    return this.applicationsService.updateStatus(Number(id), updateStatusDto, {
+      os, browser, updatedBy: Number(updatedBy), ip, updatedAt: new Date()
```

ESLint

[error] Replace `·browser,·updatedBy:·Number(updatedBy),·ip,·updatedAt:·new·Date()` with `⏎······browser,⏎······updatedBy:·Number(updatedBy),⏎······ip,⏎······updatedAt:·new·Date(),`

(prettier/prettier)